pkgs/development/tools/minipro/unstable.nix

```
@@ -0,0 +1,49 @@
+{ stdenv, installShellFiles, which, fetchgit, git, pkg-config, libusb1 }:
+stdenv.mkDerivation rec {
+  pname = "minipro";
+  version = "0.4dev";
```

Version naming convention for unstable git revs is `unstable-<date>`.
```suggestion
  version = "unstable-2020-04-18";
```